presets/ragengine/main.py

```diff
@@ -80,7 +80,12 @@ def list_indexes():
         raise HTTPException(status_code=500, detail=str(e))
 
 @app.get("/indexes/{index_name}/documents", response_model=ListDocumentsResponse)
-async def list_documents_in_index(index_name: str):
+async def list_documents_in_index(
```

Comment:
Can you add detailed examples in the change description or code comments about how to use this API? E.g., in which cases, limit/offset/max_text_length should be used?

Reply:
done added docstrings in fastapi format